extension/parquet/parquet_extension.cpp

```diff
@@ -660,9 +613,17 @@ class ParquetScanFunction {
 		auto &file_list = result->file_list;
 		file_list.InitializeScan(result->file_list_scan);
 
-		result->multi_file_reader_state = bind_data.multi_file_reader->InitializeGlobalState(
-		    context, bind_data.parquet_options.file_options, bind_data.reader_bind, file_list, bind_data.types,
-		    bind_data.names, input.column_indexes);
+		if (bind_data.reader_bind.schema.empty()) {
+			auto global_columns = MultiFileReaderColumn::ColumnsFromNamesAndTypes(bind_data.names, bind_data.types);
```

Comment:
^ Actually we cant, because MultiFileReaderColumnDefinition can be nested, we can't cleanly represent that internally as just a vector<LogicalType> types and vector<string> names